packages/input-select/src/Select.js

```diff
@@ -148,7 +148,10 @@ class Select extends React.Component {
         ) : (
           <Item
             label={item.label}
-            id={this.props.id && `${this.props.id}-item-${item.label}`}
+            id={
+              this.props.id &&
+              `${this.props.id}-item-${item.label.replace(/\s/g, "")}`
```

Comment:
How about we use `item.value`? It's less probable of having spaces, so we could simply use it here  without the `replace`.

This component already has some performance issues. Once I used it for a country selector and it got janky with about 200 items. Using regex in every item render could make it even less performant.

I'm usually against premature optimization, but in this case, as it will be used by QA, if we decide to change it later, it will break their tests.

Reply:
Since I am documenting all created ids, I agree, it's better to be by value.